mappings/net/minecraft/world/chunk/IChunk.mapping

```diff
@@ -1,7 +1,9 @@
-CLASS bpz
+CLASS bpz net/minecraft/world/chunk/IChunk
```

Comment:
We don't use I-prefixes... but then there's IWorld for lack of a better name.

Reply:
Yeah, I saw.. and I really do not like I prefixes either, but since Chunk was already taken, I didn't know of a better name to call it. ChunkBase, maybe?

Reply:
If it's getter-only, ChunkView. I think we have another ChunkView but it can be renamed?

Reply:
It's not read only, that's ReadOnlyChunk :P

This is the base interface for chunks, which the other classes like EmptyChunk, Chunk, ReadOnlyChunk implement

Reply:
Rename bpz to Chunk and Chunk to something else. Maybe WorldChunk?